units/time_series_update.go

```diff
@@ -57,7 +58,7 @@ func NewUpdateTimeSeriesJob(series model.UnanalyzedPerformanceSeries) amboy.Job
 		series.Arguments,
 	)
 	j.SetID(fmt.Sprintf("%s.%s", baseID, time.Now().UTC()))
-	j.SetScopes([]string{baseID})
+	j.SetScopes([]string{baseID, strings.Join(series.Measurements, ",")})
```

Comment:
What's the purpose of this change?

Reply:
Does it matter that series.Measurements might have variable order?

Reply:
This is to prevent race conditions where new rollups are processed async from the same perf result but with different sets of measurements (usually one will be a subset of the other). If this happens, the job with the fuller set of measurements may not run since the scope is less selective without adding the measurements to it and the `analysis.processed_at` field on the perf result document would get set to a time greater than that of the `rollups.processed_at` field preventing the backfill aggregation from picking up the perf result. 

Adding this to the job scope enable two jobs with the same base ID but different measurements to get enqueued simultaneously—we will do some extra work by sending some measurements to SPS twice, but that is better than not sending them at all. I also mitigated the duplicate sending in the perf results RPC service code by enqueueing the FTDC rollups job/time-series-updated job when `CloseMetrics` is called.

Reply:
The measurements are stored as arrays in the DB so order wouldn't be variable.

Reply:
Thanks for the explanation.